components/molecule/selectPopover/src/styles/index.scss

```diff
@@ -101,4 +101,27 @@ $base-class: '.sui-MoleculeSelectPopover ';
       transform: rotate(-90deg) !important;
     }
   }
+
+  &-overlay {
+    @each $name, $type in $molecule-select-popover-overlay-types {
+      &--#{'' + $name} {
+        $bgc: map-get($type, bgc);
+        background-color: $bgc;
+        content: '';
+        height: 100%;
+        left: 0;
+        opacity: 0;
+        pointer-events: none;
+        position: absolute;
+        top: 0;
+        transition: opacity 0.25s ease-in;
+        width: 100%;
+
+        &.is-open {
```

Comment:
don not need this class, assign directly to the upper cuz portal will remove the element from the dom

Reply:
DONE!